src/server.py

```diff
@@ -34,7 +34,8 @@
         ]
     },
     {"type": "section", "text": {"type": "mrkdwn", "text": ""}},
-    {"type": "context", "elements": [{"type": "mrkdwn", "text": "This slackbot was created by Eddy Hintze"}]}
+    {"type": "context", "elements": [{"type": "mrkdwn", "text": "This slackbot was created by Eddy Hintze and "
```

Comment:
Nice try price, this is not going to fly

Reply:
I think its important that people know